### DISCUSSION TITLE
Add kind to graph_multi_asset

### DISCUSSION THREAD

**Discussion by JordyForNow** [dagster-io/dagster#25783] [Q&A] [answered]
Hey how can i define the "kinds" on the output of a graph_multi_asset?

**Comment by garethbrickman** [comment]
[AssetOut ](https://docs.dagster.io/_modules/dagster/_core/definitions/asset_out#AssetOut)currently has support for tags but not kinds. Created a feature request for it: https://github.com/dagster-io/dagster/issues/25796

**Comment by benpankow** [comment, marked as answer]
Hi Jordy, as of `1.9.2` you're now able to [construct an `AssetOut` from a spec](https://docs.dagster.io/_apidocs/assets#dagster.AssetOut.from_spec) which gives you more control over properties like `kinds`:

```python
spec = AssetSpec(key="my_asset", kinds={"python"})

@graph_multi_asset(outs={"my_asset": AssetOut.from_spec(spec)})
def my_graph_multi_asset():
    ...
```
